recipe/build.sh

```diff
@@ -18,7 +18,11 @@ build_linux()
 
     mv _output/bin/hyperkube $PREFIX/bin
     pushd $PREFIX/bin
-    ./hyperkube  --make-symlinks
+
+    exes=(kube-apiserver kube-controller-manager kube-proxy kube-scheduler kubectl kubelet)
+    for exe in ${exes[@]}; do
+        ln -s ./hyperkube $exe
+    done
```

Comment:
This is the source of why hyperkube lost --make-symlinks:
  - https://github.com/kubernetes/kubernetes/issues/69896#issuecomment-430398309

Thanks @dims for providing me the link.